Fortran/gfortran/regression/DisabledFiles.cmake

```diff
@@ -1791,6 +1791,30 @@ file(GLOB SKIPPED_FILES CONFIGURE_DEPENDS
   pr93524.f90
   public_private_module_3.f90
   static_linking_1.f
+
+  # ----------------------------------------------------------------------------
+  #
+  # These files are only intended to be run on AArch64, but we don't currently
+  # process the target attribute, so these are disabled everywhere. When the
+  # DejaGNU target attribute is handled correctly, these should be removed from
+  # here.
+  pr101158.f90
+  pr88833.f90
+  pr98974.F90
+
+  # ----------------------------------------------------------------------------
+  #
+  # These tests have a -J flag but the build system adds a -J of its own and
+  # exactly one is allowed. If the build system is changed, these can be removed
+  # from here.
```

Comment:
Yes. 

It has to do with the way the gfortran tests are built. There are several tests which generate .mod files with the same name which causes race conditions in parallel builds. To work around this, there is a unique working directory for each test and I ended up adding a `-J` flag when building the tests. There might be a way to do it without requiring a `-J` flag, but I haven't gotten around to looking into it yet.

Reply:
I see. As `-J` was already used before, is this another case of tests that worked by accident?

Reply:
I think, in this case, the flags were not being passed while building the test at all because the DejaGNU annotation containing it was not being parsed correctly.

Reply:
Ok. I took a look at the include tests below and saw that they expect some warnings to be emitted (or omitted), which is a kind of test that we don't support anyway.

Reply:
Yes. 

You have identified several limitations with the way we are using the gfortran tests. One of the improvements that can be made is to map gfortran warnings to equivalent flang warnings where possible and check for those. Currently, warnings are completely ignored. 

We do check for the presence of an error when a test is expected to produce one, but we do not check if it is an "equivalent" error. That can be improved as well.